lib/form/deposit.js

```diff
@@ -127,7 +108,10 @@ export default class DepositForm {
     getPercentFeeDiscount(token) {
         if (this._serviceFee[token] === undefined || this._serviceFee[token] === null) return 0;
         if (this._feeDiscounts[token] === undefined || this._feeDiscounts[token] === null) return 0;
```

Comment:
Just notice a wrong condition check here, when retrieving non-existent keys the result will be `undefined`, not `null`.